.github/workflows/elixir.yml

```diff
@@ -0,0 +1,29 @@
+name: Elixir CI
+
+on:
+  push:
+    branches:
+      - '*'
+  pull_request:
+    branches:
+      - '*'
+
+jobs:
```

Comment:
Let's format this a bit better, like [this](https://github.com/inclojure-org/inclojure-org.github.io/blob/trunk/.github/workflows/pages.yml).